bootstrap.d/gui-apps.yml

```diff
@@ -0,0 +1,33 @@
+packages:
+  - name: swaybg
+    source:
+      subdir: ports
+      git: 'https://github.com/swaywm/swaybg'
+      branch: 'master'
+      commit: '3cb81c3573036a3606a4c7a1cedbe3e814a22773'
```

Comment:
Why are we building from a commit? There are releases for swaybg that we can use right?